compose/container.py

```diff
@@ -193,7 +193,7 @@ def log_driver(self):
     @property
     def has_api_logs(self):
         log_type = self.log_driver
-        return not log_type or log_type in ('json-file', 'journald')
+        return not log_type or log_type in ('json-file', 'journald', 'local')
```

Comment:
Slightly related; looks like the description here also needs updating; https://github.com/docker/compose/blob/4bc1cbc32accce59acddece7314eab6929eefe42/compose/container.py#L211

Reply:
Thanks for the tip. Updated the comment.